_drafts/how-to-contribute-to-open-source-projects-as-a-non-coder.md

```diff
@@ -0,0 +1,150 @@
+
+3. **Have a knack for organizing perhaps?**
+You can suggest issues labels and close open issues to keep the repository clean. 
+
+4. **Do you enjoy assisting others?**
+Moderate discussion boards and respond to questions on forums like Microsoft Tech Community,  Reddit, and Stack Overflow.
+
+5. **Teaching?**
+Write how-to and educational tutorials, as well as supporting materials. You can also create YouTube videos about the software.
+
+6. **Advocating?**
+Organize project-related workshops or meetups. You could motivate others by telling them about your personal experiences.
+
+7. **Join testing and insider programs:** Finding and reporting bugs, and providing honest feedback on a project can increase its success rate and provide a better user experience.
+
+So no, you don't have to code to contribute. While this is by no means an exhaustive list, it should be enough to get you started.
+
+# Benefits of contributing to open-source projects
+
+![Benefits of contributing to open source projects](https://cdn.hashnode.com/res/hashnode/image/upload/v1634906376850/1b1HfYy7-.png)
+
+While there are usually no direct monetary benefits to contributing, there are other ways you as a contributor can benefit:
+
+- Upskilling and the opportunity to improve on existing skills.
+- Creating a portfolio and personal achievement.
+- Meeting like-minded people and networking.
+- Understanding the application development and maintenance cycles.
+- Community recognition.
+- Self-recognition as an open-source contributor.
+- Recommendation, referral, and increased job opportunities. Here is a [testimony](https://www.freecodecamp.org/news/cracking-google-season-of-docs-2020/)
+- Mentorship and knowledge sharing.
+- Community building.
+- **Compensation:** There are rarely any direct monetary incentives, however, programs such as [Google Season of Docs](https://developers.google.com/season-of-docs), [Google Summer of Code](https://summerofcode.withgoogle.com/), [Hyperledger Mentorship Program](https://wiki.hyperledger.org/display/INTERN), [Outreachy](https://www.outreachy.org/), and the [Open Collective bounty program](https://docs.opencollective.com/help/contributing/development/bounties) pay you to contribute.
+
+# Open Source Projects
+
+![where can i find oen source projects](https://cdn.hashnode.com/res/hashnode/image/upload/v1634909151136/IR4gF5koo.png)
+
+Today, there are a lot of open-source projects you can contribute to and it is up to you to choose the ones that interest you. Below are a few open-source projects you can choose from; take your time to go through each resource and see which one appeals to you.
+
+- [AWS](https://aws.amazon.com/blogs/aws/aws-documentation-is-now-open-source-and-on-github/?utm_source=feedburner&utm_medium=email&utm_campaign=Feed%3A+AmazonWebServicesBlog+%28Amazon+Web+Services+Blog%29)
```

Comment:
umm there is some tracking code here. lol

Reply:
code? just links

Reply:
guess he took the link from somewhere and didn't remove it , @Bennykillua fix pls

Reply:
Oh fixed. thanks @shockz09 @Posandu